src/OpenTelemetry.Exporter.OneCollector/Logs/OneCollectorLoggerProviderBuilderExtensions.cs

```diff
@@ -0,0 +1,174 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using Microsoft.Extensions.Configuration;
+using OpenTelemetry.Exporter.OneCollector;
+using OpenTelemetry.Internal;
```

Comment:
`using OpenTelemetry.Internal;` -- this looks weird ❓

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/Shared/Guard.cs ?